tmail-backend/tmail-third-party/openpaas/src/main/java/com/linagora/tmail/OpenPaasModule.java

```diff
@@ -97,4 +98,9 @@ public ReactorRabbitMQChannelPool provideReactorRabbitMQChannelPool(
         channelPool.start();
         return channelPool;
     }
+
+    @ProvidesIntoSet
+    SimpleConnectionPool.ReconnectionHandler provideReconnectionHandler(@Named(OPENPAAS_INJECTION_KEY) OpenPaasContactsConsumer contactsConsumer) {
+        return new OpenPaasContactsConsumerReconnectionHandler(contactsConsumer);
+    }
```

Comment:
not so sure about this, i tried to do the same as what exists.

should i add some kind of
```
@Override
    protected void configure() {
        Multibinder<SimpleConnectionPool.ReconnectionHandler> reconnectionHandlerMultibinder = Multibinder.newSetBinder(binder(), SimpleConnectionPool.ReconnectionHandler.class);
        reconnectionHandlerMultibinder.addBinding().to(OpenPaasContactConsumerReconnectionHandler.class);
    }
    ```